backend/src/email/routes/tests/email-transactional.routes.test.ts

```diff
@@ -427,14 +449,19 @@ describe(`${emailTransactionalRoute}/send`, () => {
       .spyOn(FileExtensionService, 'hasAllowedExtensions')
       .mockResolvedValue(false)
 
+    await EmailFromAddress.create({
+      email: user.email,
+      name: 'Agency ABC',
+    } as EmailFromAddress)
+
```

Comment:
@stanleynguyen just curious, do you think I should shift this to `beforeEach`, or leaving it here is OK

technically not required for the test to pass (with the latest change), but good to leave it in so it's clear what the "control variable" is?